lib/dolly/request.rb

```diff
@@ -1,19 +1,19 @@
 module Dolly
   module Request
-    def set_namespace name
-      @namespace = name
+    def set_db_namespace(name)
```

Comment:
this is a breaking change, why is the change being done?

Reply:
set_namespace makes me understand that the model will set the document namespace, the part of `user/` not choose which database the model will connect to